draft-link-v6ops-464xlaton.xml

```diff
@@ -166,72 +175,35 @@ Therefore, the node SHOULD enable CLAT as soon as network support for PLAT is de
-To achieve that, the node SHOULD generate a random checksum-neutral CLAT address every time.
-</t>
-</section>
 
 <section anchor="v4addr">
 <name>
 CLAT IPv4 Addresses
 </name>
 <t>
-There are two different cases of IPv4 usage in 464XLAT deployments:
+There are two different models in 464xlat deployments:
 </t>
 <ul>
 <li>
 <t>
-Wireline network architecture, as defined in Section 4.1 of <xref target="RFC6877"/>. In that case, the node performing CLAT functions also extends the network downstream and provides network connectivity services to other connected systems.
+A dedicated prefix model, which Section 4.1 of <xref target="RFC6877"/> calls "wireline network architecture".
+In that case, the node performing CLAT functions also extends the network downstream and provides network connectivity services to other connected systems.
 Those systems can be physical (e.g. various clients connected to a CPE router), or logical (e.g. virtual systems running on a node, while the host system acts as a router and performs CLAT). 
 In all those cases, systems behind the CLAT node usually use <xref target="RFC1918"/> addresses.
 </t>
 </li>
 <li>
 <t>
-Wireless network architecture, as defined in Section 4.2 of <xref target="RFC6877"/>.
-In that case, the CLAT node provides an IPv4 address and the default route to the local node's network stack only.
+A single-address model, which section 4.2 of <xref target="RFC6877"/> calls "wireless network architecture".
+In that case, the CLAT instance provides an IPv4 address and the default route to the local node's network stack only.
 It should be noted that <xref target="RFC6877"/> implies that the second deployment scenario is limited to 3GPP cases, while currently it is also deployed in other types of networks, such as enterprise and public Wi-Fi networks, where hosts (as mobile phones, laptops and desktops) use CLAT to provide connectivity to IPv4-only local applications.
 Despite the word "wireless" in the name, this architecure is applicable for wired networks as well (e.g. desktops or servers using wired connections).
 </t>
 </li>
 </ul>
 <t>
-In the latter case, the CLAT node needs a single IPv4 address only. The node providing CLAT functions to local applications SHOULD use IPv4 addresses from the dedicated 192.0.0.0/29 range (<xref target="RFC7335"/>), reserved for IPv4 continuity solutions including but not limited to 464XLAT.
+In the latter case, the CLAT instance needs a single IPv6 and a single IPv4 address only.
+The node providing CLAT functions to local applications SHOULD use IPv4 addresses from the dedicated 192.0.0.0/29 range (<xref target="RFC7335"/>), reserved for IPv4 continuity solutions including but not limited to 464XLAT.
 If the node runs multiple CLAT instances (see <xref target="multihomed"/>), the node SHOULD use different local IPv4 addresses for each CLAT instance.
 This means that it might not be possible to run more than 8 CLAT instances per node.
```

Comment:
```suggestion
This means that it is not possible to run more than 8 CLAT instances per node.
```

We may want to provide a mechanism for exceeding 8 CLAT instances per node, but as written, the normative requirements prevent this.